test/t/unit/test_unit_dequote.py

```diff
@@ -38,7 +40,8 @@ def test_5_brace(self, bash, functions):
 
     def test_6_glob(self, bash, functions):
         output = assert_bash_exec(bash, "__tester 'a?b'", want_output=True)
-        assert output.strip() == "<a b><a$b><a&b><a'b>"
+        items = sorted(re.findall(r"<[^>]*>", output))
```

Comment:
It's not immediately clear to me why this is needed on macOS and not on others, could we elaborate in a comment?

Reply:
Yeah I'm not sure what's up with that. I just checked and LC_COLLATE seems to be set to C. I'll need to look into that